portfolio.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homework 2</title>
+  </head>
+  <body>
+    <header class="header">
+      <a href="/" class="logo">WebStudio</a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link">Студія</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Портфоліо</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Контакти</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="contacts">
+        <a href="mailto:info@devstudio.com" class="contacts__link"
+          >info@devstudio.com</a
+        >
+        <a href="tel:+380961111111" class="contacts__link">+38 096 111 11 11</a>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="portfolio">
+        <ul class="portfolio__categories categories">
+          <li class="categories__item">
+            <button class="categories__button">
+              Усі
+            </button>
```

Comment:
Always include **`type`** attrubute in **`button`** tag
```
<button class="categories__button" type="button">
  Усі
</button>
```

all buttons **`must`** have it